README.md

```diff
@@ -1093,8 +1093,31 @@ The ECS CLI supports creating, running, inspecting and stopping tasks defined by
 #### ecs-cli local up
 
 #### ecs-cli local ps
+Once you have your task running locally, the basic command to list your task's containers is:
+ ```
+$ ecs-cli local ps
+```
+This will search for containers created from the `./task-definition.json` file (to see all available options, run `ecs-cli local ps --help`).  
+
+For example, if you'd like to list containers created from another task definition file in JSON rather than the table view, use the following command:
+```
+$ ecs-cli local ps -f ./app-task-definition.json --json 
+```
```

Comment:
Should we add actual example output here?

Reply:
Updated!